saleor/product/models.py

```diff
@@ -81,7 +81,7 @@ class ProductType(models.Model):
         'Attribute', related_name='product_variant_types', blank=True)
     is_shipping_required = models.BooleanField(default=False)
     tax_rate = models.CharField(
-        max_length=128, default=DEFAULT_TAX_RATE_NAME, blank=True,
+        max_length=128, blank=True,
```

Comment:
I think that using `DEFAULT_TAX_RATE_NAME` here and in the Product model is not problematic and we could keep it. What needs to be fixed is only the dashboard form.